Framework/CurveFitting/test/Algorithms/ConvertToYSpaceTest.h

```diff
@@ -63,9 +63,9 @@ class ConvertToYSpaceTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(-1.670937938, ySpOutX[npts / 2], 1e-08);
     TS_ASSERT_DELTA(17.99449408, ySpOutX.back(), 1e-08);
     // Y
-    TS_ASSERT_DELTA(-0.01152733, ySpOutY.front(), 1e-08);
-    TS_ASSERT_DELTA(5.56667697, ySpOutY[npts / 2], 1e-08);
-    TS_ASSERT_DELTA(-0.35141703, ySpOutY.back(), 1e-08);
+    TS_ASSERT_DELTA(0.48319776, ySpOutY.front(), 1e-08);
```

Comment:
I'm a little confused as to why these values change. `ConvertToYSpace` doesn't use random numbers does it?